tests/extensions/extra/tables.txt

```diff
@@ -80,3 +80,13 @@ Lists are not tables
 
  - this | should | not
  - be | a | table
+
+Add tests for issue #449
+
+Odd backtics | Even backtics
```

Comment:
Just a note for your future PRs: you could also squash the two commits into a single one by using `git commit -a --amend` (or `git add` && `git commit --amend`). Sorry for not being so precise in my previous reply.